### DISCUSSION TITLE
Showing more than 25 listings per page in web UI

### DISCUSSION THREAD

**Discussion by key88sf** [sidekiq/sidekiq#6412] [Q&A] [answered]
Is there any way to set the "per-page" size in the UI? For example:

`https://mysite.com/sidekiq/retries?page=2`  shows only 25 jobs per page.  I couldn't find a query param which lets me show 100 per page, etc...  Is there one?

Thanks.

**Comment by mperham** [comment, marked as answer]
Try `?page=2&count=100`

**Reply by key88sf** [reply]
Works! Might want to document somewhere for others to find it too? I couldn't find it in the Wiki, etc. Thank you!

**Reply by mperham** [reply]
It's right here for people to find in the future. 😎

**Reply by key88sf** [reply]
Heh fair enough 👍
